spyder/plugins/configdialog.py

```diff
@@ -938,6 +940,49 @@ def setup_page(self):
         sbar_layout.addLayout(cpu_memory_layout)
         sbar_group.setLayout(sbar_layout)
 
+        # --- Screen resolution Group (hidpi)
+        screen_resolution_group = QGroupBox(_("Screen resolution"))
+        screen_resolution_bg = QButtonGroup(screen_resolution_group)
+
+        normal_radio = self.create_radiobutton(
+                                _("Normal"),
+                                'normal_screen_resolution',
+                                button_group=screen_resolution_bg)
+        auto_scale_radio = self.create_radiobutton(
+                                _("Enable auto high DPI scaling"),
+                                'high_dpi_scaling',
+                                button_group=screen_resolution_bg,
+                                tip=_("Set this for high DPI displays"))
+
+        custom_scaling_radio = self.create_radiobutton(
+                                _("Set a custom high DPI scaling"),

```

Comment:
We should also add a link to Qt documentation that explains this, like http://doc.qt.io/qt-5/highdpi.html